Tools/Scripts/webkitpy/w3c/test_importer.py

```diff
@@ -121,8 +121,13 @@ def format(self, record):
 # FIXME: We should decide whether we want to make this specific to web-platform-tests or to make it generic to any git repository containing tests.
 def parse_args(args):
     description = """
-To import a web-platform-tests test suite named xyz, use: 'import-w3c-tests web-platform-tests/xyz'.
-To import a web-platform-tests suite from a specific folder, use 'import-w3c-tests xyz -s my-folder-containing-web-platform-tests-folder'"""
+To import a web-platform-tests test suite named xyz, use:
+    import-w3c-tests --tip-of-tree web-platform-tests/xyz
+
+To import a web-platform-tests suite from a specific local checkout out of web platform tests:
+   1. Your checkout must be a folder called "web-platform-tests".
```

Comment:
```suggestion
   1. Your working tree must be in a directory called "web-platform-tests".
```

(per https://support.apple.com/en-gb/guide/applestyleguide/apsg7af4f5d0/web#apd11f2025f4c0b4, "In developer materials, use directory"; git's docs mostly use "working tree" and not checkout)

Reply:
"working tree" is confusing IMO. That's not colloquial terminology. I think "checkout" is more appropriate because one does "git **checkout** git:some-repo"

Reply:
yes I have difficulties to understand the new sentence proposed here. 

What about:

1. Your local WPT copy must be in a directory called  "web-platform-tests".

Reply:
To make it clearer

```suggestion
   1. Your local WPT copy must be in a directory called "web-platform-tests".
```

Reply:
To the point that @marcoscaceres is making, the change is to try to make the help easy to understand for anyone. And indeed colloquial is then probably more important in this case.

Reply:
I like @karlcow wording.

Reply:
done.